File: CS/ExportToXLSX/Form1.cs
using System;
using System.Diagnostics;
using System.Windows.Forms;
using DevExpress.XtraPrinting;
// ...

namespace ExportToXLSX {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            // A path to export a report.
            string reportPath = "c:\\Test.xlsx";

            // Create a report instance.
            XtraReport1 report = new XtraReport1();

            // Get its XLS export options.
            XlsxExportOptions xlsxOptions = report.ExportOptions.Xlsx;

            // Set XLS-specific export options.
            xlsxOptions.ShowGridLines = true;
            xlsxOptions.TextExportMode = TextExportMode.Value;
            xlsxOptions.ExportHyperlinks = true;
            xlsxOptions.SheetName = "My Sheet";

            // Export the report to XLS.
            report.ExportToXlsx(reportPath);

            // Show the result.
            StartProcess(reportPath);
        }

        // Use this method if you want to automaically open
        // the created XLS file in the default program.
        public void StartProcess(string path) {
            Process process = new Process();
            try {
                process.StartInfo.FileName = path;
                process.Start();
                process.WaitForInputIdle();
            }
            catch { }
        }

    }
}

File: VB/ExportToXLSX/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Diagnostics
Imports System.Windows.Forms
Imports DevExpress.XtraPrinting
' ...

Namespace ExportToXLSX
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' A path to export a report.
			Dim reportPath As String = "c:\Test.xlsx"

			' Create a report instance.
			Dim report As New XtraReport1()

			' Get its XLS export options.
			Dim xlsxOptions As XlsxExportOptions = report.ExportOptions.Xlsx

			' Set XLS-specific export options.
			xlsxOptions.ShowGridLines = True
			xlsxOptions.TextExportMode = TextExportMode.Value
			xlsxOptions.ExportHyperlinks = True
			xlsxOptions.SheetName = "My Sheet"

			' Export the report to XLS.
			report.ExportToXlsx(reportPath)

			' Show the result.
			StartProcess(reportPath)
		End Sub

		' Use this method if you want to automaically open
		' the created XLS file in the default program.
		Public Sub StartProcess(ByVal path As String)
			Dim process As New Process()
			Try
				process.StartInfo.FileName = path
				process.Start()
				process.WaitForInputIdle()
			Catch
			End Try
		End Sub

	End Class
End Namespace